src/binding/c/info_api.txt

```diff
@@ -18,6 +18,10 @@ MPI_Info_delete:
     .desc: Deletes a (key,value) pair from info
     .skip: initcheck
     .extra: NotThreadSafe
+{ -- error_check --
+    MPIR_ERR_CHKANDJUMP((info == MPI_INFO_ENV), mpi_errno, MPI_ERR_INFO,
+                        "**info");
```

Comment:
Let's make the error message a bit more precise -- how about `**infoenv: MPI_INFO_ENV is read-only`?

Reply:
Done.